batchflow/tests/named_expr_test.py

```diff
@@ -94,16 +99,16 @@ def test_apply_parallel_p(named_expr, src):
     else:
         assert (b.images == b.masks).all()
 
-
+@pytest.mark.parametrize('p_type', P_OPTIONS)
 @pytest.mark.parametrize('named_expr', P_NAMED_EXPRS)
-def test_inbatch_parallel_p(named_expr):
+def test_inbatch_parallel_p(p_type, named_expr):
```

Comment:
this test checks only that the pipeline doesn't fail

It does not check that NE's are calculated properly

I think these tests need thorough rework